legend-engine-xt-graphQL-pure/src/main/resources/core_external_query_graphql/deprecated/generation.pure

```diff
@@ -38,7 +38,13 @@ function <<doc.deprecated>> meta::external::query::graphQL::generation::generate
 {
    $config.scopeElements->filter(e| !$e->instanceOf(Package))->map(e|assert($e->instanceOf(Type),' GraphQL generation of ' + $e->type().name->toOne()+ ' is not currently supported. Tried to generate GraphQL for '+ $e->elementToPath()) ;);
    let scopeElements = $config.allPackageScopeElements()->filter(p|$p->instanceOf(Class))->cast(@Class<Any>);
-   $scopeElements->map(c| $config->meta::external::query::graphQL::generation::generateGraphQLFromPure($c));
+   $scopeElements->map(c| $config->meta::external::query::graphQL::generation::generateGraphQLFromPure($c))->concatenate(
```

Comment:
Couple of comments ...

1/ Let's not generate a custom scalar type definition if the corresponding Pure type is not in the generation scope 

2/ Separately, we should not be inlining logic to compute a GraphQLOutput in generateGraphQL. Let's move into a separate function

Reply:
done